stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/ChannelUserRead.kt

```diff
@@ -2,8 +2,17 @@ package io.getstream.chat.android.client.models
 
 import java.util.Date
 
+/**
+ * Information about how many messages are unread in the channel by a given user
```

Comment:
Every sentence should end with punctuation. Check [internal guidelines](https://github.com/GetStream/InternalAndroidGuidelines#kdoc-comments) for  more details.
This applies to both description and properties description